src/staveline.ts

```diff
@@ -154,7 +154,9 @@ export class StaveLine extends Element {
   protected font: FontInfo;
   protected first_indices!: number[];
   protected last_indices!: number[];
-  protected notes: StaveLineNotes;
+
+  // Initialized by this.setNotes(notes) in the constructor.
+  protected notes!: StaveLineNotes;
```

Comment:
The linter does not complain?

Reply:
Surprisingly, I am not getting any eslint complaints when I add a ! 
I need to investigate...